addons/event_sale/models/sale_order.py

```diff
@@ -74,18 +75,9 @@ def _update_registrations(self, confirm=True, registration_data=None):
                 registration = {}
                 if registration_data:
                     registration = registration_data.pop()
-                # TDE CHECK: auto confirmation
```

Comment:
I would keep this comment, unless you are sur auto confirm is always correctly taken into account (still something to check before saas-6).